### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by AbdollahRaafat**
API First I need to choose an API to base the development of the webapp on. The API should allow me to get numeric values based on a parameter. We recommend that I choose an API that is open or needs an API key. If yI choose an API that require authentication, I should implement it on my own.
 I should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
- Colors (select one main color for your website).
- Typography: font face, size and weight.
- Layout: composition and space between elements.
 Home page:
- When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.
 Details page:
- In the details page, the webapp retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
 Technical requirements:
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- You should filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project should be deployed and accessible online (add a link to the online version of your app to your README file)
 Project documentation
Once I have finished the development of the project, I should record a video presenting the features of the project I built. It is a video with a maximum length of 5 minutes. The content of the video should include:
a description of the project.
a demo of the project features.
I should also highlight some interesting piece of code or something you built that I'm very proud of.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+- Tailwind CSS
+
+- Framer Motion
+
+- ReactIcons
+
+## Live Demo
+
+<a href="https://effervescent-puffpuff-a0122d.netlify.app/">COVID-19 Metrics Data</a>
```
Comment (ShahierNashaat) [inline README.md]: - [ ] Kindly notice that the live demo link is not working.

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+
+## Live Demo Video
+
+<a href="https://www.loom.com/share/623fb8fe2a3b4dda919e4e5056fc11d0">COVID-19- Video Demo</a>
```
Comment (ShahierNashaat) [inline README.md]: - [ ] Kindly notice that the video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @AbdollahRaafat, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- All tests are passing ✔️
- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] Kindly delete the snap shots and run the tests again to make all of the test passes.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (mikethreels) [inline README.md]: - [ ] as the previous CR already mentioned the video should contain a highlight about a piece of the code you find interesting or are proud of making.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Abdo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @AbdollahRaafat, 

Good job on completing the required changes! Your project now is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**